frontend/src/components/BigPopup/CommentSection/CommentForm.tsx

```diff
@@ -38,6 +38,7 @@ const CommentForm = () => {
 
   const onSendComment: SubmitHandler<CommentCheck> = ({ commentText }) => {
     const commentBody: CommentType<string> = {
+      _id: userId,
```

Comment:
I didn`t understand for what we need here _id if we have author? author same id as _id or not ?

Reply:
because whose write in interface "authorId: string | undefined" and if i use authorId i have a conflict: "Argument of type 'string | undefined' is not assignable to parameter of type 'string'.
  Type 'undefined' is not assignable to type 'string'", and i thought that i create new variable for my task, and don't change authorId type, maybe it uses somewhere, but if i remove ' | undefined' then i can use authorId